modules/ROOT/pages/queries-aggregations/filtering.adoc

```diff
@@ -103,7 +103,108 @@ query {
 Spatial types use numerical filtering differently and they also have additional options. 
 See xref:filtering.adoc#_filtering_spatial_types[Filtering spatial types] for more information.
 
-==== Spatial type filtering
+[source, javascript, indent=0]
+----
+const { Neo4jGraphQL } = require("@neo4j/graphql");
+const neo4j = require("neo4j-driver");
+
+const typeDefs = `
+    type User @node {
+        name: String
+    }
+`;
+
+const driver = neo4j.driver(
+    "bolt://localhost:7687",
+    neo4j.auth.basic("username", "password")
+);
+
+const features = {
+    filters: {
+        String: {
+            LT: true,
+            GT: true,
+            LTE: true,
+            GTE: true
+        }
+    }
+};
+
+const neoSchema = new Neo4jGraphQL({ features, typeDefs, driver });
+----
+
+== RegEx matching
+
+The filter `_MATCHES` is also available for comparison of `String` and `ID` types.
```

Comment:
this file/section has been updated (recently, 10th of october) in 6.x:
https://github.com/neo4j/docs-graphql/blob/6.x/modules/ROOT/pages/queries-aggregations/filtering.adoc#regex-matching

not sure why the changes didn't make it here :thinking:

Reply:
Good catch! let me check it!